yt_dlp/extractor/piaulizaportal.py

```diff
@@ -44,27 +99,9 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, video_id)
 
-        player_data = self._download_webpage(
-            self._search_regex(
-                r'<script [^>]*\bsrc="(https://player-api\.p\.uliza\.jp/v1/players/[^"]+)"',
-                webpage, 'player data url'),
-            video_id, headers={'Referer': 'https://ulizaportal.jp/'},
-            note='Fetching player data', errnote='Unable to fetch player data')
-
-        formats = self._extract_m3u8_formats(
-            self._search_regex(
-                r'["\'](https://vms-api\.p\.uliza\.jp/v1/prog-index\.m3u8[^"\']+)', player_data,
-                'm3u8 url', default=None),
-            video_id, fatal=False)
-        m3u8_type = self._search_regex(
-            r'/hls/(dvr|video)/', traverse_obj(formats, (0, 'url')), 'm3u8 type', default=None)
-
-        return {
-            'id': video_id,
-            'title': self._html_extract_title(webpage),
-            'formats': formats,
-            'live_status': {
-                'video': 'is_live',
-                'dvr': 'was_live',  # short-term archives
-            }.get(m3u8_type, 'not_live'),  # VOD or long-term archives
-        }
+        player_data_url = self._search_regex(
+            r'<script [^>]*\bsrc="(https://player-api\.p\.uliza\.jp/v1/players/[^"]+)"',
+            webpage, 'player data url')
+        return self.url_result(
+            player_data_url, url_transparent=True,
```

Comment:
To not have to check all extractors for a matching regex again
```suggestion
            player_data_url, PIAULIZAPortalAPIIE, url_transparent=True,
```